index.html

```diff
@@ -0,0 +1,208 @@
+                    <div class="activity-info">Listen to speakers from various countries about the recent advancements in electromagnetic theory.</div>
+                </div>
+                <div class="program-item">
+                    <img src="./images/exhibition_icon.png" class="icon-main-program">
+                    <div class="activity">Exhibition</div>
+                    <div class="activity-info">Appreciate new and old theories applying the concept of Faraday's tube by renowed scientists
+                    </div>
+                </div>
+        
+                <div class="program-item">
+                    <img src="./images/forum_icon.png" class="icon-main-program">
+                    <div class="activity">Forum</div>
+                    <div class="activity-info">Have the time to share your thoughts and opinions with experts for each topic.</div>
+                </div>
+        
+                <div class="program-item">
+                    <img src="./images/workshop_icon.png" class="icon-main-program">
+                    <div class="activity">Workshop</div>
+                    <div class="activity-info">Try creating your own work using your own eperiments rather than just watching at it.</div>
+                </div>
+                
+                <div class="program-item">
+                    <img src="./images/ignite.png" class="icon-main-program">
+                    <div class="activity">Ignite</div>
+                    <div class="activity-info">Get opportunities to network with CC affiliates around the world, also after the summit.</div>
+                </div>                
+                
+            </div>
+            
+            
+            <button class="orange-button">
+                Join CC Global Summit 2015
+            </button>
+
+            <div> <a href="#"> See the whole program </a></div>
+            
+    
+        </div>
+            
+        
+        <div class="title-speakers">
+            <h2 class="text-style"> Featured Speakers</h2>
+            <div class="line"></div>
+        </div>
+                    
+
+        <div class="speakers"> </div>
+        <div class="speakers-bis"> </div>
+        <div class="more">More <i id="more-icon" class="bi bi-chevron-down" style="color: red; "></i> </div>        
+        <div class="less">Less <i id="less-icon" class="bi bi-chevron-up" style="color: red; "></i> </div>
+
+
+        <div class="sponsors-desktop">
+            <div class="sponsor-1"> Partner</div>
+            <div class="line"></div>
+            <div class="container-items">
+                <div > <img src="./images/sponsor-2.png"></div>
+                <div> <img src="./images/sponsor-3.png"></div>
+                <div> <img src="./images/sponsor-4.png"></div>
+                <div> <img src="./images/sponsor-5.png"></div>
+                <div> <img src="./images/sponsor-6.png"></div>            
+            </div>  
+        </div>
```

Comment:
- Kindly use `ul` for the list of sponsors and add `alt` for each of your images for accessibility👍